From 25cf4db5575192fd306b7a74d496d1e10d8cfd3e Mon Sep 17 00:00:00 2001
From: Mart Raudsepp <leio@gentoo.org>
Date: Fri, 27 Nov 2020 22:47:30 +0200
Subject: [PATCH] [X11] Fix build with disabled OpenGL

The added PlatformDisplayX11::visual() method requires symbols from
X11/Xutil.h, which wasn't added as an explicit include with only
EGL headers indirectly pulled it in with standard build options.
---
 Source/WebCore/platform/graphics/x11/PlatformDisplayX11.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Source/WebCore/platform/graphics/x11/PlatformDisplayX11.cpp b/Source/WebCore/platform/graphics/x11/PlatformDisplayX11.cpp
index 8fe3d3e92c88..2b2805446aad 100644
--- a/Source/WebCore/platform/graphics/x11/PlatformDisplayX11.cpp
+++ b/Source/WebCore/platform/graphics/x11/PlatformDisplayX11.cpp
@@ -30,6 +30,7 @@
 
 #if PLATFORM(X11)
 #include <X11/Xlib.h>
+#include <X11/Xutil.h>
 #include <X11/extensions/Xcomposite.h>
 #if PLATFORM(GTK)
 #include <X11/extensions/Xdamage.h>
-- 
2.20.1

